webcams/straja.php

```diff
@@ -0,0 +1,13 @@
+<div class="webcam">
+    <span>Plecare Gondola</span>
+    <div class="box">
+        <iframe src="https://www.trans-alpina.ro/camera/plecare.jpg?counter=1583695628169" frameborder="0"></iframe>
```

Comment:
Straja are not working as it's forbidden. 
Check the approach I followed for Poiana Brasov via `image.php`.

Compare the referal sent via Request Header by the original website which embeds the webcam.

Reply:
![image](https://user-images.githubusercontent.com/12177811/76170240-b5194480-6188-11ea-82e1-a50d541b4dbc.png)

Protip: you should extend `image.php` and get `?referal=X` key.
Combined with an array of `$referals = [...]` in `init.php` you then can get and use `$referal` correctly in `image.php`.